ontology/yaml/resources/subfields/subfields.yaml

```diff
@@ -242,6 +243,7 @@ descriptor:
   transducer: "A device that converts energy from one form to another (e.g., sound energy to electrical energy in a microphone, electrical energy to sound energy in a speaker, etc.)."
   ultraviolet: "A short wavelength type of light that is beyond the visible spectrum. It is damaging to DNA and cells, and is commonly used to sterilize surfaces."
   unoccupied: "State of being outside of scheduled (time of day or week) run-time parameters."
+  up: "The direction of a spatial movement, opposite of down."
```

Comment:
Can you please move these fields (down, up) to the `measurement_descriptor` section (they are similar to open/closed)

Reply:
done